web/src/settings_users.js

```diff
@@ -263,11 +263,17 @@ function bot_info(bot_user_id) {
 }
 
 function get_last_active(user) {
-    const last_active_date = presence.last_active_date(user.user_id);
+    const presence_row = user_setting_presence_info.get(String(user.user_id));
```

Comment:
I don't think the `String` conversion is needed.

Reply:
@sahil839, I tried that, but it always results in "Unknown," which means the `presence_row` value isn’t being fetched correctly without using a string.